packages/paste-website/src/components/site-wrapper/site-header/SiteHeaderMobile.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {HamburgerToggle} from './HamburgerToggle';
+import {SiteHeaderLogo} from './SiteHeaderLogo';
+import {MobileNavigation} from './SiteHeaderMobileNavigation';
+
+interface SiteHeaderMobileProps {
+  searchValue: string;
+  onSearchChange: (newSearchValue: string) => void;
+}
+
+const SiteHeaderMobile: React.FC<SiteHeaderMobileProps> = (/* {searchValue, onSearchChange} */) => {
```

Comment:
Commented out arguments?

Reply:
Yeah for this PR. They need to be passed through but I'm not using them yet. Mobile search in a follow up PR.